explainer.md

```diff
@@ -344,54 +239,30 @@ an author could specify this relationship programmatically instead:
 ```js
 const input = comboBox.shadowRoot.querySelector("input");
 const optionList = comboBox.querySelector("custom-optionlist");
-input.accessibleNode.activeDescendant = optionList.accessibleNode;
+input.activeDescendantElement = optionList.firstChild;
 ```
 
 This would allow the relationship to be expressed naturally.
 
-#### AOM and ARIA
+##### Precedence between string-based and element-based relationship properties
 
-While AOM and ARIA both affect the computed accessible properties of a node,
-and have the same vocabulary,
-they are separate interfaces.
-
-##### No reflection
-
-ARIA does not reflect to AOM:
-
-```html
-<div id="clickBtn" role="button">Click here</div>
-```
+Element-based relationship properties take precedence over 
+(reflected) string-based properties,
+while not affecting the DOM attribute value:
 
 ```js
-console.log(clickBtn.accessibleNode.role);   // null, not "button"
-```
+// Reflects to "aria-labelledby" DOM attribute
+button.ariaLabelledBy = "id1";
 
-And AOM does not reflect to ARIA:
+// DOM result: <button aria-labelledby="id1">
 
-```html
-clickBtn.accessibleNode.role = "link";
+// Takes precedence; does not affect reflected attribute value
+el.ariaLabelledByElements = [el2, el3];
 
-console.log(clickBtn.getAttribute("role"));  // Still "button"
+// DOM remains unchanged: <button aria-labelledby="id1">
 ```
 
-##### AOM takes precedence
-
-In the case where and AOM Accessible Property and the corresponding ARIA attribute have different values,
-the AOM property takes precedence.
-
-For example, in the case in the previous section,
-`clickBtn` would have a computed role of `"link"`
-even though its ARIA role is of `"button"`.
-
-##### Reasoning
 
-The reasons for these decisions are discussed in [#60](https://github.com/WICG/aom/issues/60), but in summary
-AOM is meant to be a superset of ARIA in terms of its capabilities, so
-it makes sense for AOM Accessible Properties to always supercede ARIA.
-Authors who wish to use AOM due to its greater expressiveness or
-smaller overhead (relative to setting a DOM attribute) won't have to
-worry about a DOM attribute overriding it.
 
 #### Computed accessible properties
```

Comment:
This section still has a handful of examples using `accessibleNode`

Reply:
Oops 🙁 Deleted.